lib/repl.js

```diff
@@ -203,6 +204,9 @@ function REPLServer(prompt,
     }
   }
 
+  const preview = options.terminal &&
```

Comment:
How about set `preview` option to `this.preview` such as `this.useColors`, `this.useGlobal`, etc below for consistency ?
```suggestion
  this.preview = options.terminal &&
```

Reply:
Exposing the variable should not provide any new information for the user. Since I do not see any use case, I'd rather not expose this as additional variable on the instance.